samples/GiraffeRazorSample/GiraffeRazorSample.fsproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
It might be worth mutli-targeting as did for `Giraffe` and support these TFMs: https://github.com/giraffe-fsharp/Giraffe/blob/master/src/Giraffe/Giraffe.fsproj#L12
Although I could see how that could be more trouble than it's worth. I'd imagine we can target `netcoreapp3.0` only and it work smoothly on `3.0` and `3.1` (needs proving though).

Reply:
The problem is this fellow:
https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation